modules/dev/src/tests/exec.ts

```diff
@@ -0,0 +1,30 @@
+import { TestNamespace, TestObject } from "../types";
+import { experimental, fs } from "@replit/extensions";
+import { assert } from "chai";
+
+const { exec } = experimental;
+
+const tests: TestObject = {
+  exec: async () => {
+    await exec.exec("touch test_text_file.txt");
+    const res = await fs.readFile("test_text_file.txt");
+
+    assert.isObject(res);
+
+    if ("content" in res) {
+      assert.isString(res.content);
+    } else {
+      throw new Error("Expected a string");
+    }
+
+    // Cleanup
+    await fs.deleteFile("test_text_file.txt");
```

Comment:
I find it weird to test library methods with library methods, but i guess it's the best we can do for now

other than that the tests are great

Reply:
tests are still very much a WIP.  Will write a bunch more today.